desktop/src/main/java/bisq/desktop/main/overlays/windows/EditCustomExplorerWindow.java

```diff
@@ -0,0 +1,198 @@
+        listView.setItems(FXCollections.observableArrayList(availableExplorers));
+        headLine = coin + " " + Res.get("settings.preferences.editCustomExplorer.headline");
+    }
+
+    public BlockChainExplorer getEditedBlockChainExplorer() {
+        return new BlockChainExplorer(nameInputTextField.getText(),
+                txUrlInputTextField.getText(), addressUrlInputTextField.getText());
+    }
+
+    public void show() {
+
+        width = 1000;
+        createGridPane();
+        addHeadLine();
+        addContent();
+        addButtons();
+
+        urlInputValidator = new UrlInputValidator();
+        txUrlInputTextField.setValidator(urlInputValidator);
+        addressUrlInputTextField.setValidator(urlInputValidator);
+        nameInputTextField.setValidator(new LengthValidator(1, 50));
+
+        actionButton.disableProperty().bind(createBooleanBinding(() -> {
+                    String name = nameInputTextField.getText();
+                    String txUrl = txUrlInputTextField.getText();
+                    String addressUrl = addressUrlInputTextField.getText();
+
+                    // Otherwise we require that input is valid
+                    return !nameInputTextField.getValidator().validate(name).isValid ||
+                            !txUrlInputTextField.getValidator().validate(txUrl).isValid ||
+                            !addressUrlInputTextField.getValidator().validate(addressUrl).isValid;
+                },
+                nameInputTextField.textProperty(), txUrlInputTextField.textProperty(), addressUrlInputTextField.textProperty()));
+
+        applyStyles();
+        display();
+    }
+
+    @Override
+    protected void createGridPane() {
+        gridPane = new GridPane();
+        gridPane.setHgap(15);
+        gridPane.setVgap(15);
+        gridPane.setPadding(new Insets(64, 64, 64, 64));
+        gridPane.setPrefWidth(width);
+        ColumnConstraints columnConstraints1 = new ColumnConstraints();
+        ColumnConstraints columnConstraints2 = new ColumnConstraints();
+        columnConstraints1.setPercentWidth(45);
+        columnConstraints2.setPercentWidth(55);
+        gridPane.getColumnConstraints().addAll(columnConstraints1, columnConstraints2);
+    }
+
+    private void addContent() {
+        Label mlm = addMultilineLabel(gridPane, rowIndex++, Res.get("settings.preferences.editCustomExplorer.description"), 0);
+        GridPane.setColumnSpan(mlm, 2);
+        GridPane.setMargin(mlm, new Insets(40, 0, 0, 0));
+
+        Button button = new Button(">>");
+        button.setOnAction(e -> {
+            BlockChainExplorer blockChainExplorer = listView.getSelectionModel().getSelectedItem();
+            if (blockChainExplorer != null) {
+                nameInputTextField.setText(blockChainExplorer.name);
+                txUrlInputTextField.setText(blockChainExplorer.txUrl);
+                addressUrlInputTextField.setText(blockChainExplorer.addressUrl);
+            }
+        });
```

Comment:
Please use our AutoTooltipButton or JFXButton to match the design of all other buttons.

Reply:
:+1: Done!